frontend/src/components/Banner.tsx

```diff
@@ -11,23 +11,7 @@ export type BannerProps = {
   type?: "promo" | "warning" | "info";
   title?: string;
   illustration?: ReactNode;
-  cta?: {
-    target: string;
-    content: string;
-    onClick?: () => void;
-    gaViewPing?: Parameters<typeof useGaViewPing>[0];
-  };
-  ctaLargeButton?: {
-    target: string;
-    content: string;
-    onClick?: () => void;
-    gaViewPing?: Parameters<typeof useGaViewPing>[0];
-  };
-  btn?: {
-    content: string;
-    onClick?: () => void;
-    gaViewPing?: Parameters<typeof useGaViewPing>[0];
-  };
+  cta?: BannerCtaProps;
```

Comment:
I refactored the props to `<Banner>` a bit to unify the `cta`, `ctaLarge` and `btn` props into a single `cta` prop.

- To get the `ctaLargeButton` behaviour, pass the same properties to `cta`, and additionally `size: "large"`.
- To get the `btn` behaviour, pass the same properties to `cta`. The lack of a `target` turns it into a `<button>` rather than an `<a>`.